doc-content/enterprise-only/cases/case-management-stage-activation-and-completion-conditions-proc.adoc

```diff
@@ -3,24 +3,24 @@
 
 Stages can be triggered by a start node, intermediate node, or manually using an API call.
 
-You can configure stages with both activation and completion conditions using free-form Drools rules, the same way that milestone completion conditions are configured. For example, in the IT Orders sample project, the `Milestone 2: Order shipped` completion condition (`org.kie.api.runtime.process.CaseData(data.get("shipped") == true)`) can also be used as the completion condition for the `Order delivery` stage represented here:
+You can configure stages with both activation and completion conditions using free-form Drools rules, the same way that milestone completion conditions are configured. For example, in the *IT_Orders* sample project, the `Milestone 2: Order shipped` completion condition (`org.kie.api.runtime.process.CaseData(data.get("shipped") == true)`) can also be used as the completion condition for the `Order delivery` stage represented here:
 
-.IT Orders project stages example
-image::cases/itorders-stages.png[IT Orders - stages]
+.IT_Orders project stages example
+image::cases/itorders-stages.png[IT_Orders - stages]
 
-Activation conditions can also be configured using a free-form Drools rule to configure the `AdHocActivationCondition` property to activate a stage. 
+Activation conditions can also be configured using a free-form Drools rule to configure the `AdHocActivationCondition` property to activate a stage.
 
-.Prerequisite 
+.Prerequisite
 * You have created a case definition in the {CENTRAL} process designer.
 * You have added an ad hoc subprocess to the case definition that is to be used as a stage.
 
 .Procedure
 . With the stage selected on the case design canvas, click image:cases/3898.png[] to open the *Properties* panel on the right.
-. Open the `AdHocActivationCondition` property editor to define an activation condition for the start node. For example, `autostart: true` to make the stage automatically activated when a new case instance is started. 
+. Open the `AdHocActivationCondition` property editor to define an activation condition for the start node. For example, `autostart: true` to make the stage automatically activated when a new case instance is started.
```

Comment:
For example, set the AdHoc (or some other verb)

Reply:
Done